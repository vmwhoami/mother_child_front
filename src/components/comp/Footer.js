import React from 'react';
import css from '../../css/nav.module.css';
import {
  Gplus, Pinterest, Twitter, FaceBook,
} from './Svgs';

const Footer = () => (
  <div className={css.footer}>
    <div className={css.social}>
      <a href="https://twitter.com/" target="blank">
        {' '}
        <Twitter style={css.svg} />
      </a>
      <a href="https://www.facebook.com/" target="blank">
        <FaceBook style={css.svg} />
      </a>
      <a href="https://myaccount.google.com/" target="blank">
        <Gplus style={css.svg} />
      </a>
      <a href="https://www.pinterest.com/" target="blank">
        <Pinterest style={css.svg} />
      </a>

    </div>

    <span className={css.allrights}>
      <a href="https://www.vitalimelnic.tech">
        &copy;
        {` ${(new Date()).getFullYear()}  by Vitalie Melnic.`}
      </a>
    </span>

  </div>
);

export default Footer;
