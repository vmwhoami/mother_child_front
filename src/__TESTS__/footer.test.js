import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/component/Footer';

it('renders Footer correctly', () => {
  const tree = renderer.create(<Footer />);
  expect(tree).toMatchSnapshot();
});
