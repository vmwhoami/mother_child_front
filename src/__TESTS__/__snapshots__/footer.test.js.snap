// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer correctly 1`] = `
<div
  className="footer"
>
  <div
    className="social"
  >
    <a
      href="https://twitter.com/"
      target="blank"
    >
       
      <svg
        className="svg"
        version="1.1"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Twitter
        </title>
        <path
          d="M17.316 6.246c0.008 0.162 0.011 0.326 0.011 0.488 0 4.99-3.797 10.742-10.74 10.742-2.133 0-4.116-0.625-5.787-1.697 0.296 0.035 0.596 0.053 0.9 0.053 1.77 0 3.397-0.604 4.688-1.615-1.651-0.031-3.046-1.121-3.526-2.621 0.23 0.043 0.467 0.066 0.71 0.066 0.345 0 0.679-0.045 0.995-0.131-1.727-0.348-3.028-1.873-3.028-3.703 0-0.016 0-0.031 0-0.047 0.509 0.283 1.092 0.453 1.71 0.473-1.013-0.678-1.68-1.832-1.68-3.143 0-0.691 0.186-1.34 0.512-1.898 1.861 2.285 4.644 3.787 7.781 3.945-0.064-0.277-0.097-0.564-0.097-0.861 0-2.084 1.689-3.773 3.774-3.773 1.086 0 2.067 0.457 2.756 1.191 0.859-0.17 1.667-0.484 2.397-0.916-0.282 0.881-0.881 1.621-1.66 2.088 0.764-0.092 1.49-0.293 2.168-0.594-0.506 0.758-1.146 1.422-1.884 1.953z"
        />
      </svg>
    </a>
    <a
      href="https://www.facebook.com/"
      target="blank"
    >
      <svg
        className="svg"
        version="1.1"
        viewBox="0 0 32 32"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          FaceBook
        </title>
        <path
          d="M19 6h5v-6h-5c-3.86 0-7 3.14-7 7v3h-4v6h4v16h6v-16h5l1-6h-6v-3c0-0.542 0.458-1 1-1z"
        />
      </svg>
    </a>
    <a
      href="https://myaccount.google.com/"
      target="blank"
    >
      <svg
        className="svg"
        version="1.1"
        viewBox="0 0 36 28"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Gplus
        </title>
        <path
          d="M22.453 14.266c0 6.547-4.391 11.188-11 11.188-6.328 0-11.453-5.125-11.453-11.453s5.125-11.453 11.453-11.453c3.094 0 5.672 1.125 7.672 3l-3.109 2.984c-0.844-0.812-2.328-1.766-4.562-1.766-3.906 0-7.094 3.234-7.094 7.234s3.187 7.234 7.094 7.234c4.531 0 6.234-3.266 6.5-4.937h-6.5v-3.938h10.813c0.109 0.578 0.187 1.156 0.187 1.906zM36 12.359v3.281h-3.266v3.266h-3.281v-3.266h-3.266v-3.281h3.266v-3.266h3.281v3.266h3.266z"
        />
      </svg>
    </a>
    <a
      href="https://www.pinterest.com/"
      target="blank"
    >
      <svg
        className="svg"
        version="1.1"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Pinterest
        </title>
        <path
          d="M8.617 13.227c-0.526 2.754-1.167 5.394-3.068 6.773-0.586-4.162 0.861-7.287 1.534-10.605-1.147-1.93 0.138-5.812 2.555-4.855 2.975 1.176-2.576 7.172 1.15 7.922 3.891 0.781 5.479-6.75 3.066-9.199-3.485-3.538-10.146-0.083-9.326 4.982 0.199 1.238 1.478 1.613 0.511 3.322-2.231-0.494-2.897-2.254-2.811-4.6 0.138-3.84 3.449-6.527 6.771-6.9 4.201-0.471 8.144 1.543 8.689 5.494 0.613 4.461-1.896 9.293-6.389 8.945-1.218-0.095-1.728-0.699-2.682-1.279z"
        />
      </svg>
    </a>
  </div>
  <span
    className="allrights"
  >
    <a
      href="https://www.vitalimelnic.tech"
    >
      Â©
       2021  by Vitalie Melnic.
    </a>
  </span>
</div>
`;
